media-libs/libdmtx/libdmtx-0.7.4.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Library to read and write Data Matrix barcodes"
+DESCRIPTION="libdmtx is a software library that enables programs to read and \
+write Data Matrix (or DataMatrix) 2D barcodes. It runs on several platforms, \
+and can be accessed by multiple languages."
+HOMEPAGE="http://libmtx.sourceforge.net/"
+COPYRIGHT="2005-2011 Mike Laughton"
+LICENSE="libdmtx v1.0"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/libdmtx/libdmtx-$portVersion.tar.gz"
```

Comment:
Please change SOURCE_URI to:
```
SOURCE_URI="https://downloads.sourceforge.net/libdmtx/libdmtx-$portVersion.tar.gz"
```
(the difference is only the "s" in "download**s**")
This is because "download.sourceforge.net" is just a CNAME (i.e. a canonical name, a.k.a. DNS alias) for "downloads.sourceforge.net", so it is better to use the main domain name, although both work.